src/Guest.js

```diff
@@ -22,17 +22,29 @@ class Guest extends Hotel {
     })
   }
 
-  getGuestTotalSpent() {
-    return this.rooms.reduce((totalSpent, room) => {
-      this.bookings.forEach(booking => {
+  calculateGuestTotalSpent() {
+    const total = this.pastBookings.reduce((totalSpent, booking) => {
+      this.rooms.forEach(room => {
         if (booking.roomNumber === room.number) {
-          totalSpent += room.costPerNight
+          totalSpent += (room.costPerNight * 100)
```

Comment:
Instead of multiplying and dividing by 100, try "element.toFixed(2)" so the 2 restricts it to two decimal places